gn3/computations/traits.py

```diff
@@ -17,20 +12,20 @@ def fetch_trait(dataset, trait_name: str) -> dict:
         "trait_name": trait_name
     }
 
-    trait_data = get_trait_sample_data(dataset, trait_name)
+    trait_data = get_trait_sample_data(dataset, trait_name, database)
```

Comment:
**suggestion:** Let your naming be consistent(I'm referring to how you call the db connections).